src/Summoner/Project.hs

```diff
@@ -12,36 +12,26 @@ import NeatInterpolation (text)
 import System.Info (os)
 import System.Process (readProcess)
 
-import Summoner.Ansi (infoMessage, skipMessage, successMessage, warningMessage)
+import Summoner.Ansi (infoMessage, successMessage, warningMessage)
 import Summoner.Config (Config, ConfigP (..))
 import Summoner.Default (currentYear, defaultGHC)
 import Summoner.License (License (..), customizeLicense, githubLicenseQueryNames, licenseNames)
 import Summoner.Process ()
 import Summoner.ProjectData (CustomPrelude (..), Decision (..), ProjectData (..), parseGhcVer,
                              showGhcVer, supportedGhcVers)
-import Summoner.Question (checkUniqueName, choose, query, queryDef, queryManyRepeatOnFail)
+import Summoner.Question (checkUniqueName, choose, chooseYesNo, chooseYesNoBool, falseMessage,
+                          query, queryDef, queryManyRepeatOnFail, trueMessage)
 import Summoner.Template (createStackTemplate)
 import Summoner.Tree (traverseTree)
 
-import qualified Data.Text as T
 import qualified Data.Text.IO as T
 import qualified Universum.Unsafe as Unsafe
 
 decisionToBool :: Decision -> Text -> IO Bool
 decisionToBool decision target = case decision of
-    Yes -> trueMessage  target
-    Nop -> falseMessage target
-    Idk -> do
-        ch <- choose ("Add " <> target <> "?") ["y", "n"]
-        case ch of
-            "y" -> trueMessage  target
-            "n" -> falseMessage target
-            _   -> error "Impossible happened"
-
-trueMessage, falseMessage :: Text -> IO Bool
-trueMessage  target = True  <$ successMessage (T.toTitle target <> " will be added to the project")
-falseMessage target = False <$ skipMessage (T.toTitle target <> " won't be added to the project")
-
+    Yes -> True  <$ trueMessage  target
+    Nop -> False <$ falseMessage target
```

Comment:
I see that you do `True <$` and `False <$` for every use site of `trueMessage` and `falseMessage`. Earlier `trueMessage` was written like `True <$ successMessage ... smth ...`. Maybe I'm missing something. But is there reason to make return types of `trueMessage` and `falseMessage` be `IO ()` instead of previous `IO Bool`?

Reply:
The reason is that I'm using it in the `chooseYesNo` where return type is `IO a` not `IO Bool`.. But probably I just need to write separate functions for messages itself..

Reply:
@vrom911 I see. But in `chooseYesNo` function you wrote like this:

```haskell
trueMessage target >> yesDo
```

Since you use `>>` it doesn't matter what is the return type of `trueMessage`. Probably you wrote just `true` and `falseMessage` earlier and saw type errors, but after you refactored, compiler didn't warn you that you can return types back. I think it should work if you make type as `IO Bool`.